tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -10,19 +10,24 @@ internal static class Program
     private static unsafe void Main(string[] args)
     {
         Add(1, 2);
-		int tempInt = 0;
-		(*(&tempInt)) = 9;
+        Shift(1, 2);
```

Comment:
Would it be possible to test the result of this similar to the other tempInt check? (I have an idea of how to do more reasonable-looking testing. I'll try to do it soon.)